Package: acpica-unix
Severity: important
Version: 20060912-2
Tags: patch


Hi,

the current version fails to build on GNU/kFreeBSD.

It needs small tweak to define uintptr_t, see bellow.

It would also be nice if you can ask upstream
to include this change.

Thanks in advance

                         Petr


---
 include/platform/acenv.h |    3 +++
 1 file changed, 3 insertions(+)

Index: b/include/platform/acenv.h
===================================================================
--- a/include/platform/acenv.h	2010-06-26 14:48:00.495143360 +0900
+++ b/include/platform/acenv.h	2010-06-26 14:50:54.395126529 +0900
@@ -182,6 +182,9 @@
 
 
 /*! [Begin] no source code translation */
+#if defined(__GLIBC__)
+#include <stdint.h>
+#endif
 
 /******************************************************************************
  *
